app/models/registration/pages/DeedOfVariation.scala

```diff
@@ -14,24 +14,28 @@
  * limitations under the License.
  */
 
-package mapping
+package models.registration.pages
 
 import models.{Enumerable, WithName}
+import viewmodels.RadioOption
 
 sealed trait DeedOfVariation
 
 object DeedOfVariation extends Enumerable.Implicits {
 
-  case object DeedOfVariation extends WithName("Previously there was only an absolute interest under the will") with DeedOfVariation
-  case object ReplacedWill extends WithName("Replaced the will trust") with DeedOfVariation
-  case object AdditionToWill extends WithName("Addition to the will trust") with DeedOfVariation
+  case object ReplacedWill extends WithName("ReplacedWill") with DeedOfVariation
```

Comment:
Changing these values will not conform to the expected values for submission.
This will fail when submitted I think?